theme.json

```diff
@@ -97,6 +97,63 @@
 			],
 			"units": ["%", "px", "em", "rem", "vh", "vw"]
 		},
+		"typography": {
+			"defaultFontSizes": false,
+			"fluid": true,
+			"fontSizes": [
+				{
+					"fluid": false,
+					"name": "Extra Small",
+					"size": "14px",
+					"slug": "x-small"
+				},
+				{
+					"fluid": {
+						"max": "18px",
+						"min": "18px"
+					},
+					"name": "Small",
+					"size": "1rem",
+					"slug": "small"
+				},
+				{
+					"fluid": {
+						"max": "22px",
+						"min": "18px"
+					},
+					"name": "Medium",
+					"size": "1.38rem",
+					"slug": "medium"
+				},
+				{
+					"fluid": {
+						"max": "32px",
+						"min": "28px"
+					},
+					"name": "Large",
+					"size": "1.75rem",
+					"slug": "large"
+				},
+				{
+					"fluid": {
+						"max": "42px",
+						"min": "38px"
+					},
+					"name": "Extra Large",
+					"size": "3rem",
```

Comment:
This looks inconsistent with the min/max defined for XL (`3rem` is `48px`, where the min/max are `38px` and `42px` - In the figma the size is `38px` for XL)

Reply:
Ah ok, thanks for the clarification @beafialho